index.js

```diff
@@ -0,0 +1,56 @@
+const form = document.querySelector('form');
+const bookList = document.getElementById('book-list');
+const bookArr = [];
+
+const checkItem = localStorage.getItem('bookArrObj');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const obj = {
+    title: document.getElementById('book_title').value,
+    author: document.getElementById('author').value,
+  };
+
+  if (obj.title === '' || obj.author === '') {
+    e.preventDefault();
+  } else if (!checkItem) {
+    bookArr.push(obj);
+    localStorage.setItem('bookArrObj', JSON.stringify(bookArr));
+
+    location.reload();
+  } else {
+    const ArrayStored = localStorage.getItem('bookArrObj');
+    const ArrayStoredParse = JSON.parse(ArrayStored);
+    ArrayStoredParse.push(obj);
+    localStorage.setItem('bookArrObj', JSON.stringify(ArrayStoredParse));
+
+    location.reload();
+  }
+});
+
+const BookStored = JSON.parse(localStorage.getItem('bookArrObj'));
+
+function updateUi() {
+  const buttonRemove = document.createElement('button');
+  const bookLi = document.createElement('div');
+  buttonRemove.innerText = 'Remove';
+
+  for (let i = 0; i < BookStored.length; i += 1) {
+    bookLi.innerHTML += `<p>${BookStored[i].title}</br>${BookStored[i].author}</p>
+    <button>Remove</button> <hr>`;
+
+    bookList.append(bookLi);
+  }
+
+  const removeBtns = document.querySelectorAll('#book-list button');
+  removeBtns.forEach((a, i) => {
+    a.addEventListener('click', () => {
+      BookStored.splice(i, 1);
+      localStorage.setItem('bookArrObj', JSON.stringify(BookStored));
+
+      location.reload();
+    });
+  });
+}
```

Comment:
[OPTIONAL]

- [ ] Please favor the use of array methods that doesn't mutate the array, e.g. `concat, filter, slice` instead of `push, pop, splice` and so on. 
- [ ] I suggest that you use unique `id` property for each book since it would make it easier to remove books with duplicate values.